### PR TITLE
Header

### PR DESCRIPTION
**Pull request by PurpurTentakel97**
see #2 #3 #4

### CODE REVIEW

File: src/elements/header.svelte [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<header>
+	<div class="nav-wrapper">
+		<a href="#top">
+			<img
+				src = '/logo_small.png'
+				alt="The small Test-Conf logo. It shows the name on black clouds."
```
Comment (mgerhold) [inline src/elements/header.svelte:6]: Not a show-stopper, but this is inconsistently formatted. Also the usage of `"` and `'` is inconsistent.

File: src/elements/header.svelte [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+			/>
+		</a>
+		<nav>
+			<li><a href="#top">Start</a></li>
+			<li><a href="#Speaker">Vortragende</a></li>
+			<li><a href="#Sponsors">Sponsoren und Medienpartner</a></li>
+			<li><a href="#Shedule">Ablaufplan</a></li>
+			<li><a href="/login">Login</a></li>
+		</nav>
```
Comment (mgerhold) [inline src/elements/header.svelte:15]: As Pedder pointed out in your stream, I think `<li>` elements must be contained in either `<ul>` or `<ol>` tags.

File: static/GnuolaneBk-Regular.otf
Comment (mgerhold) [inline static/GnuolaneBk-Regular.otf:1]: The font files have to be removed.

**Review by mgerhold — CHANGES_REQUESTED** [review]

File: src/elements/header.svelte (hunk 1 above)
Comment (PurpurTentakel97) [reply src/elements/header.svelte:6]: done

File: src/elements/header.svelte (hunk 2 above)
Comment (PurpurTentakel97) [reply src/elements/header.svelte:15]: done

File: static/GnuolaneBk-Regular.otf
Comment (PurpurTentakel97) [reply static/GnuolaneBk-Regular.otf:1]: done

**Review by mgerhold — APPROVED** [review]

**Pull request merged**